js/basic-cad-text-electron.js

```diff
@@ -0,0 +1,231 @@
+function addText() {
+  var fontsize = $('#fontsize').val();
+  var font = $("#font").val().replace(/\+/g, ' ');
+  // split font into family and weight
+  font = font.split(':');
+  // set family on paragraphs
+  var string = $("#texttorender").val()
+  // console.log('font-family: ', font[0], " size: ", fontsize, " String: " + string)
```

Comment:
Can you remove any commented out code that wasn't used?

Reply:
Yes, the majority of the commented out code is left from the original plugins.  Do you want me to delete any commented out code that I come across? Or just the ones that I added/left?

Reply:
ok, I didn't realize that was a copy of a file. You can leave the commented code as is.